<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShaderDialog</class>
 <widget class="QDialog" name="ShaderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="VertexShader">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>175</height>
       </rect>
      </property>
      <attribute name="label">
       <string>VertexShader</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="VertexShaderText"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="vertexOpenFile">
         <property name="text">
          <string>Open file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="vertexSaveFile">
         <property name="text">
          <string>Save file</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="GeomShader">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>175</height>
       </rect>
      </property>
      <attribute name="label">
       <string>GeomShader</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="geomOpenFile">
         <property name="text">
          <string>Open file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="geomSaveFile">
         <property name="text">
          <string>Save file</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="GeomShaderText"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="FragmentShader">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>175</height>
       </rect>
      </property>
      <attribute name="label">
       <string>FragmentShader</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QPushButton" name="fragmentOpenFile">
         <property name="text">
          <string>Open file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="fragmentSaveFile">
         <property name="text">
          <string>Save file</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="FragmentShaderText"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="execute">
     <property name="text">
      <string>execute shader</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
